Userland/Libraries/LibGUI/TextEditor.cpp

```diff
@@ -1003,6 +1003,30 @@ void TextEditor::keydown_event(KeyEvent& event)
         return;
     }
 
+    if (event.ctrl() && event.key() == KeyCode::Key_Slash) {
+        if (m_highlighter != nullptr) {
```

Comment:
This implementation looks like it wouldn't play nicely with an action in e.g. the TextEditor app or HackStudio. How are other edit commands handled in those apps with regards to their GUI::TextEditor widgets? For e.g. putting them in the Edit menu bar or as a right click action.